prepare-to-load-test-application-on-openshift.md

```diff
@@ -0,0 +1,65 @@
+---
+  title: Prepare to load test an application on OpenShift
+
+  description:  Describe what to do before you load test an application
+
+  keywords: application profiling, profiling, network traffic, bandwidth, resiliency, HA, high availability, failover, testing, guidelines, silver, gold, OpenShift, load test, application
+
+  page purpose: Describes the steps needed to load test an application in OpenShift. Provides further information on what developers need to do before and to schedule a test.
+
+  audience: developer
+
+  author: Jonathan Bond
+
+  content owner: Cailey Jones
+---
+# Prepare to load test an application on OpenShift
+Follow these guidelines to load test ministry applications hosted in the Silver and Gold cluster of the OpenShift 4 platform.
+
+## Requirements
+
+As OpenShift is a shared platform, the Platform Services team wants to make sure you're successful when you load test your application. You have to meet all the following requirements:
+
+1. The test must take place after 5:30pm on a weekday
+1. The maximum number of **concurrent sessions** during load testing must be restricted to 40,000
+1. There must be a Platform Team member present during the test
+
+The first two requirements help make sure that the test doesn't create any unnecessary impact on other applications running on the cluster.
+
+It's unlikely this will cause issues. OpenShift isolates each namespace effectively, though there can be unexpected outcomes to some tests which impact other shared services.
+
+Following these requirements helps avoid unexpected outcomes.
+
+**Note**: The Silver cluster of OpenShift 4 Platform includes 6 router nodes which together can handle 60,000 concurrent active connections. Browsers and other clients usually hold a connection open for some time and make multiple requests in the same connection or use a long running connection for things like websockets. To keep the cluster stable and healthy, only two-thirds of the overall routing capacity can be targeted by the application load test.
+
+Having a Platform Services team member present benefits your team and the platform. The Platform Services team wants keep an eye on the platform during the test so they can react in case the test causes something unexpected to shared services.
```

Comment:
```suggestion
Having a Platform Operations team member present benefits your team and the platform. The Platform Services team wants keep an eye on the platform during the test so they can react in case the test causes a negative impact  on the operations of the Openshift cluster, the Platform services or other applications hosted on that cluster.
```